tests/RobotFramework/tasks/E2E_Tutorials/Connect_to_c8y.robot

```diff
@@ -33,11 +33,23 @@ Create the certificate
 
     # You can then check the content of that certificate.
     ${output}=    Execute Command    sudo tedge cert show    # You can then check the content of that certificate.
```

Comment:
This check is a duplicate of `tests/RobotFramework/tests/cumulocity/registration/cumulocity_ca.robot` and is not executed by the CI.

- @reubenmiller Are the tests under `tests/RobotFramework/tasks` useful?
- If not it would be better to remove all of them.

Reply:
Sorry I missed this question. I've approached your PR (https://github.com/thin-edge/thin-edge.io/pull/3495) to remove these unused tasks. Good suggestion!